### PR TITLE
Update documentation for symlinking package folder during development

### PR DESCRIPTION
**Pull request by gurbirkalsi**

### CODE REVIEW

**Comment by gurbirkalsi** [comment]
Resolves issue #11

File: README.md [hunk 1]
```diff
@@ -52,34 +52,35 @@ The `tests` directory represents the testing environment for ensuring native fun
...
 
-This will create a reference to the `jschart/` directory under dependencies in `package.json`:
+Verify Local jschart Node Module Installation
 
-```
-"dependencies": {
-  "jschart": "file:jschart"
-}
+```bash
+$ readlink node_modules/jschart
```
Comment (k-rister) [inline README.md]: Do we want to suggest the use of the -e argument to readlink?  Without it I get this:

```
$ readlink node_modules/jschart
../../../../../.config/yarn/link/jschart
```

Which is just pointing to yet another symbolic link.  With -e it I get a complete resolution to the actual file:

```
$ readlink -e node_modules/jschart
/home/krister/data/code/jschart/jschart
```

**Review by k-rister — COMMENTED** [review]
Looks good otherwise.

File: README.md (hunk 1 above)
Comment (gurbirkalsi) [reply README.md]: Thanks for catching that @k-rister, just pushed updated changes.

**Review by k-rister — APPROVED** [review]
LGTM

**Pull request merged**